source/load-balancer/layer-7.rst

```diff
@@ -68,6 +68,23 @@ may also have an association to a back-end pool. Policies describe actions that
 should be taken by the load balancing software if all of the rules in the
 policy return true.
 
+L7 policy position
+^^^^^^^^^^^^^^^^^^
+
+The load balancer service maintains the relative order of the layer 7 policies
+for a given load balancer.  The policy positions are kept in a strict numbered
+sequence starting at 1.  If changes are made to the positions of a load
+balancer's policies then the service will renumber the positions to maintain the
+sequence.  It is important to remember this if using an orchestration tool to
+manage the polices as the position values of the policies when created may not
+match the positions specified which could caused your tool to mark the creation
+as a failure or lead to duplicate policies being created.
```

Comment:
This sentence is pretty run on. Should be broken up. 

"It is important to remember this if you are using an orchestration tool to manage your policies. The values of your policies positions when created may not match the positions specified in your orchestration instruction file. This could cause your orchestration tool to mark the creation as a failure and lead to duplicated policies being created."